src/ignore_directives.rs

```diff
@@ -197,14 +205,17 @@ export function deepAssign(
 target: Record<string, any>,
 ...sources: any[]
 ): // deno-lint-ignore ban-types
-object | undefined {}
+Object | undefined {}
+
+// deno-lint-ignore no-explicit-any no-empty no-debugger -- reason for ignoring
+function foo(): any {}
```

Comment:
This looks good, I have only one worry - it will be easy to make comments over 80 or 100 characters with explanation. I wonder if we could figure out a way to make these comment multi-line.

Could you try to add a test case for something like:
```
// deno-lint-ignore no-explicit-any no-empty no-debugger -- super, duper, very long
// reason for ignoring
function foo(): any {}
```

Reply:
I tried the test case you gave. it was able to be passed.
but `dlint` with real TS file did not work.

Do you want to finish multi-line issue in this PR? Or we can open a new issue for it with new PR.

P.s. A small question: Is it common to add comments under 'lint ignore' (Is it common in Node.js)? I personally have never seen it before.

Reply:
Yeah, we can do multi-line support in a follow up. Let's land it as is.